<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>545</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>545</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Copains</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Entrez deux nombres et nous vous dirons si ils sont copains</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstUserInput">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Tester</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructionLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="scdUserInput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
